homeassistant/components/enocean/utilities.py

```diff
@@ -0,0 +1,15 @@
+"""Utility methods for common ENOcean operation."""
+from typing import List
+
+
+def enocean_id_to_string(identifier: List[int]) -> str:
+    """Return a decodable string representation of an ENOcean identifier."""
+    return ":".join([f"{val:02X}" for val in identifier])
+
+
+def string_to_enocean_id(string: str) -> List[int]:
+    """Return a numeric enocean id decoded from a string."""
+    numeric_id = [int(element, 16) for element in string.split(":")]
+    if len(numeric_id) != 4:
+        raise Exception(f"Invalid enocean id string: {string}")
```

Comment:
Please create a new exception that inherits from `HomeAssistantError`.

Reply:
Custom exception created and used here.